help/en/releasenotes/current-draft-note.shtml

```diff
@@ -231,7 +231,8 @@
 
         <h4>Digitrax</h4>
             <ul>
-                <li></li>
+                <li>Updated Series 7 definitions.</li>
+                <li>Added Series 8 definitions.</li>
```

Comment:
Since we also have decoder definitions for "series 7" _accessory_ decoders (i.e. LocoNet devices), it may be appropriate to specify these in the release notes as "Mobile" decoders.

Reply:
Thanks.  Done.